ext/nnstreamer/tensor_filter/tensor_filter_qnn.cc

```diff
@@ -321,7 +321,7 @@ QnnManager::init (const char *model_path, const char *backend_path)
 
   uint32_t graph_count = 0U;
   ModelError_t mrt = compose_graphs (backend_handle_, qnn_interface_, context_handle_,
-      nullptr, 1, &graphs_info_, &graph_count, false, nullptr, QNN_LOG_LEVEL_ERROR);
+      nullptr, 0U, &graphs_info_, &graph_count, false, nullptr, QNN_LOG_LEVEL_ERROR);
```

Comment:
Thank you for PR :+1:
I have a question, I want to know the difference between 1 and 0U. :)

Reply:
It's the number of `GraphConfigInfo_t *` being passed to compose_graphs and it's nullptr here. So it should be 0.